pkg/controllers/dynakube/monitored_entities/config.go

```diff
@@ -0,0 +1,9 @@
+package monitoredentities
```

Comment:
we don't have folders with snake-case, so we shouldn't start now, to remain consistent

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3447/commits/7e1e568e4e9045f69237947d9217d7ec22f8d380